steward/src/somm_send.rs

```diff
@@ -54,6 +61,21 @@ pub async fn send_allocation(
     __send_messages(contact, delegate_key, fee, vec![msg]).await
 }
 
+pub async fn send_cork(
+    contact: &Contact,
+    cork: Cork,
+    delegate_address: String,
+    delegate_key: CosmosPrivateKey,
+    fee: Coin,
+) -> Result<TxResponse, CosmosGrpcError> {
+    let msg = MsgSubmitCorkRequest {
+        cork: Some(cork),
```

Comment:
Why is this an `Option`? There is no valid `MsgSubmitCorkRequest` that doesn't contain a `cork`.

Reply:
I agree. This is on the sommelier proto side and is how prost generates the message type for some reason.